### PR TITLE
Update main android activity name

### LINKED ISSUE #32581: Failure to run android app natively from editor
**Godot version:**
3.2.alpha
**OS/device including version:**
Arch Linux, reproduces on Android 9 and Oculus Quest
**Issue description:**
When pressing the android icon in the upper right corner to run, it fails to run on the device stating that the Intent `com.godot.GodotApp` could not be found.
**Steps to reproduce:**
1. Plug in android device, and enable USB debugging
2. Attempt to run on android device
**Minimal reproduction project:**

### PR DESCRIPTION
**Pull request by creikey**
Fixes #32581

### CODE REVIEW

File: platform/android/export/export.cpp [hunk 1]
```diff
@@ -1537,7 +1537,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		args.push_back("-a");
 		args.push_back("android.intent.action.MAIN");
 		args.push_back("-n");
-		args.push_back(get_package_name(package_name) + "/com.godot.game.GodotApp");
+		args.push_back(get_package_name(package_name) + "/org.godotengine.godot.Godot");
```
Comment (m4gr3d) [inline platform/android/export/export.cpp]: @creikey That's not correct. The new activity name for the Android template is `com.godot.game.GodotApp`. What you are changing back to is no longer valid.

**Review by m4gr3d — CHANGES_REQUESTED** [review]

**Comment by akien-mga** [comment]
I suspect @creikey is using old Android templates with a current master, hence the discrepancy.

**Comment by m4gr3d** [comment]
> I suspect @creikey is using old Android templates with a current master, hence the discrepancy.

Most likely. @creikey Follow the [new instructions](https://github.com/godotengine/godot-docs/pull/2796) to compile the Android templates, and try again.

**Comment by creikey** [comment]
After doing some investigation, it seems as if I'm using android build templates built around 2 months ago with a freshly built copy of the editor, causing several issues. Apologies for the faulty PR.

**Pull request closed without merging**